README.md

```diff
@@ -88,9 +88,9 @@ Refer to [CONTRIBUTING](./CONTRIBUTING.md) for build instructions.
 
 Nessie Iceberg's integration is compatible with Iceberg as in the following table:
 
-| Nessie version | Iceberg version | Spark version                                                        | Hive version | Flink version  | Presto version            |
-|----------------|-----------------|----------------------------------------------------------------------|--------------|----------------|---------------------------|
-| 0.43.0         | 0.14.1          | 3.1.X (Scala 2.12), 3.2.X (Scala 2.12+2.13), 3.3.X (Scala 2.12+2.13) | n/a          | 1.14.x, 1.15.x | 0.274.x, 0.275.x, 0.276.x |
+| Nessie version | Iceberg version | Spark version                                                        | Hive version | Flink version  | Presto version |
+|----------------|-----------------|----------------------------------------------------------------------|--------------|----------------|----------------|
+| 0.43.0         | 1.0.0           | 3.1.x (Scala 2.12), 3.2.x (Scala 2.12+2.13), 3.3.x (Scala 2.12+2.13) | n/a          | 1.14.x, 1.15.x | 0.276.x 0.277  |
```

Comment:
do i understand correctly, we can update this table for the already released nessie `0.43.0` because we have integration tests that confirm nessie client `0.30.0` in iceberg `1.0.0` is compatible with the nessie `0.43.0` server?

nit: isnt presto `0.277` bound to have a `0.277.x` potentially?
then we might want to use that instead for consistency

Reply:
https://github.com/projectnessie/query-engine-integration-tests/pull/49